docs/src/content/docs/resources/plugins.mdx

```diff
@@ -1,23 +1,10 @@
 ---
-title: Starlight Showcase
-description: Discover sites built with Starlight and community tools that extend Starlight!
+title: Plugins and Integrations
+description: Discover community tools that extend Starlight!
```

Comment:
Should this page description also mention "plugins" too somehow?

Reply:
Yes!